src/App.js

```diff
@@ -151,89 +152,87 @@ class App extends Component {
-                exact
-                path="/language/:language"
-                component={BrowseSkillByLanguage}
-              />
-              <Route
-                exact
-                path="/:category/:skills/:lang"
-                component={SkillListing}
-              />
-              <Route
-                exact
-                path="/:category/:skills/:lang/feedbacks"
-                component={SkillFeedbackPage}
-              />
-              <ProtectedRoute exact path="/dashboard/" component={Dashboard} />
-              <Route
-                exact
-                path="/:category/:skill/versions/:lang"
-                component={SkillVersion}
-              />
-              <Route
-                exact
-                path="/:category/:skill/compare/:lang/:oldid/:recentid"
-                component={SkillHistory}
-              />
-              <Route
-                exact
-                path="/:category/:skill/edit/:lang/:latestid/:revertid"
-                component={SkillRollBack}
-              />
-              <Route
-                exact
-                path="/:category/:skill/edit/:lang"
-                component={SkillWizard}
-              />
-              <Route
-                exact
-                path="/:category/:skill/edit/:lang/:commit"
-                component={SkillWizard}
-              />
-              <ProtectedRoute exact path="/myskills" component={Dashboard} />
-              <ProtectedRoute
-                exact
-                path="/skillWizard"
-                component={SkillWizard}
-              />
-              <ProtectedRoute
-                path="/botbuilder/botwizard"
-                component={BotBuilderWrap}
-              />
-              <ProtectedRoute path="/botbuilder" component={Dashboard} />
-              <Route exact path="/about" component={Overview} />
-              <Route exact path="/devices" component={Devices} />
-              <Route exact path="/team" component={Team} />
-              <Route exact path="/blog" component={Blog} />
-              <Route exact path="/contact" component={Contact} />
-              <Route exact path="/support" component={Support} />
-              <Route exact path="/terms" component={Terms} />
-              <Route exact path="/privacy" component={Privacy} />
-              <Route exact path="/verify-account" component={VerifyAccount} />
-              <Route exact path="/logout" component={Logout} />
-              <Route path="/admin" component={Admin} />
-              <ProtectedRoute exact path="/settings" component={Settings} />
-              <Route exact path="/*:path(error-404|)" component={NotFound} />
-            </Switch>
-            <div
-              style={
-                applyFooterStyle.includes(pathname)
-                  ? { marginTop: '230px' }
-                  : null
-              }
-            >
-              {renderFooter}
-            </div>
+            <Components>
```

Comment:
Rename it RootContainer.

Reply:
This is already defined

Reply:
I think it is already fixed for mobile and tab views.

Reply:
Rename Components to RootContainer
RootContainer -> div

Reply:
@akshatnitd Done